molecule/check_new_topology/converge.yml

```diff
@@ -0,0 +1,194 @@
+---
+
+- name: 'Change URI of core-1 instance'
+  hosts: core-1-replica
+  become: true
+  become_user: root
+  gather_facts: false
+  tasks:
+    - name: 'Set new advertise_uri'
+      set_fact:
+        config:
+          advertise_uri: 'vm1:3305'
+          http_port: 8105
+
+    - vars:
+        cartridge_force_advertise_uris_change: true
+        cartridge_scenario:
+          - configure_instance
+          - restart_instance
+          - wait_instance_started
+          - connect_to_membership
```

Comment:
Is this step really needed there? There are two versions of one member after connecting to membership?

Reply:
Yes, after this membership will have 5 instances instead of 4

Reply:
I add this info in name of task